packages/block-library/src/post-comment-content/index.php

```diff
@@ -17,8 +17,12 @@ function render_block_core_post_comment_content( $attributes, $content, $block )
 	if ( ! isset( $block->context['commentId'] ) ) {
 		return '';
 	}
-
-	return sprintf( '<div>%1$s</div>', get_comment_text( $block->context['commentId'] ) );
+	$wrapper_attributes = get_block_wrapper_attributes();
+	return sprintf(
+		'<div%1$s>%2$s</div>',
+		'' !== $wrapper_attributes ? ' ' . $wrapper_attributes : '',
```

Comment:
Same as for `post-comment-author`. Notice how `post-author` didn't care about spacing.